pkg/resources/schema.go

```diff
@@ -71,6 +71,15 @@ func SizeSchema(resource string) *schema.Schema {
 	}
 }
 
+func ManagedClusterSizeSchema(resource string) *schema.Schema {
```

Comment:
Nit. Will this attribute be shared across multiple resources in the future or is it specific to clusters? If it is specific I would favor keeping it specific to the cluster schema.

Reply:
I like that idea, just updated it accordingly!